server/src/instant/admin/routes.clj

```diff
@@ -593,6 +593,29 @@
                             room-data)]
     (response/ok {:sessions enhanced-room-data})))
 
+;; --------------- 
+;; Soft deletions 
+
+(defn soft-deleted-attrs-get [req]
+  (let [{:keys [app-id]} (req->app-id-authed! req :data/read)
+        soft-deleted-attrs (attr-model/get-soft-deleted-by-app-id
+                            (aurora/conn-pool :read)
+                            app-id)]
+    (response/ok {:soft-deleted-attrs soft-deleted-attrs})))
+
+(defn hard-delete-attrs-post [req]
```

Comment:
Why do we need to provide a hard delete to the user?

If we handle deletes in the background ourselves, then we can manage transaction sizes and pause deletes when the system is overloaded.

Reply:
Removed @dwwoelfel !